modules/RTC/RTCUtils.js

```diff
@@ -122,7 +123,10 @@ function getConstraints(um, options) {
         } else {
             // Prefer the front i.e. user-facing camera (to the back i.e.
             // environment-facing camera, for example).
-            constraints.video.facingMode = 'user';
+            // TODO: maybe use "exact" syntax if options.facingMode is defined
+            // but this probably needs to be updated with other constraints,
+            // as we currently don't use "exact" syntax anywhere
```

Comment:
1. I don't understand your comment because I don't know what "exact" means. Could you please provide the standard which defines what that is?

2. Could I please ask you to convert this comment to a proper sentence i.e. capitalization, full stop, etc?